pkg/reconciler/pipeline/controller.go

```diff
@@ -860,6 +860,93 @@ func (r *pipelineReconciler) updateDesiredState(ctx context.Context, pl *dfv1.Pi
 	}
 }
 
+func (r *pipelineReconciler) vertexResumeHandler(
+	ctx context.Context,
+	pl *dfv1.Pipeline,
+	filter vertexFilterFunc,
```

Comment:
Can we remove `filter` and `newDesiredPhase`?
Since this function is only for resuming, let's do a simple and straightforward implementation.